src/daemons/MetaDaemon.cpp

```diff
@@ -95,6 +86,25 @@ int main(int argc, char *argv[]) {
     }
     auto& localhost = hostAddrRet.value();
 
+    // folly IOThreadPoolExecutor
+    auto ioPool = std::make_shared<folly::IOThreadPoolExecutor>(FLAGS_num_io_threads);
+
+    LOG(INFO) << "Starting Meta HTTP Service";
+    nebula::WebService::registerHandler("/meta", [&localhost, &ioPool] {
+        auto client = std::make_shared<nebula::meta::MetaClient>(
```

Comment:
Typically,  we don't want a server module depend on client module.

Reply:
yep, it's strange I will use `KVStore` to replace it